core/escrow/escrow.go

```diff
@@ -52,8 +52,8 @@ func NewContract(configuration *config.Config, id string, n *core.IpfsNode, pid
 		CollateralAmount: 0,
 		WithholdAmount:   0,
 		TokenType:        escrowpb.TokenType_BTT,
-		PayoutSchedule:   0,
-		NumPayouts:       1,
+		PayoutSchedule:   payoutSchedule,
```

Comment:
don't see where you use the   int32 customize_payout_period = 12; // customized payout period in days
if this is not set, we will still use current 30 days as pay out period.
enum Schedule {
  MONTHLY = 0;
  QUARTERLY = 1;
  ANNUALLY = 2;
  CUSTOMIZED = 3;
}